spec-release-process.md

```diff
@@ -0,0 +1,85 @@
+# Specification Release Process
+
+This process governs the motion of a specification from draft through full standardization. A specification is termed a _deliverable_ as it passes through various phases from _draft deliverable_ to _accepted deliverable_, at which point it becomes a final specification.
+
+## Versioning
+
+A specification will call out a target version (such as v1.0.0), and then also include its stability marker (such as Draft, see below) to indicate progress towards the target version.
+
+For example, the glue-cose specification should be formally referenced as _glue-cose v1.0.0_. The phase of the process MAY be appended as a stability marker: glue-cose v1.0.0-AD_.
```

Comment:
Just a question, as glue-cose effort is outisde, Veraison organization, perhaps it is better to give an example from go-cose, rather than gluecose for veraison repositories